### ISSUE TITLE
What is the license Koloboke and Koloboke compiler shipped with?

### ISSUE THREAD

**Issue by szhem** [leventov/Koloboke#59]
Hi there, 

Currently I've discover that Koloboke Compile contains the following statements in META-INF/License.txt

    License for Koloboke Compile
    
    1. You can
     - Use this software for free.
     - (Re)distribute this software for free (without modification and this license text included).
    
    2. You cannot (without permission acquired from Roman Leventov at leventov@ya.ru)
     - Sell this software.
     - Repackage this software or include it into other software.
     - Modify this software.
     - Decompile this software (obtain it's source code).
     - Produce work derived from original or decompiled sources of this software.
     - Sell or distribute any work or software that is the result of one of the manipulations listed
       above.

Actually the current license agreement can be treated as **"use this lib 
for informational purposes only"** and any use within the commercial software, your own projects, and any other kind of software is forbidden.

Is it correct?

**Comment by leventov** [comment]
@szhem no, the point was that you can use Koloboke Compile as provided dependency that works during annotation processing, even in commercial software.

I had plans to truly open source Koloboke Compile (it would be copyleft then - but still, you can use it as described above even in closed-source, comm software), but don't have time unfortunately, and demand as well

**Comment by szhem** [comment]
Thanks a lot for clarification, got it.

**Comment by dingwa** [comment]
Hi,

So just to be clear with regards to the license, can I use Koloboke as a dependency in software that I create and sell?

**Comment by leventov** [comment]
@dingwa Koloboke library - yes. Koloboke Compile - only if Koloboke Compile's binary is not included into your software, i. e. it is used only at the building process for annotation processing.
